src/main/resources/application.yml

```diff
@@ -141,5 +141,5 @@ aws:
       photos: bb-good-photos
       packages: barter-bay-app-packages
     key:
-      secret: ${S3_SECRET}
-      access: ${S3_ACCESS}
+      secret: dPYZcxWQV2zuK/jQvLh5pkN9QNDS20/46ZVM2Y8n
+      access: AKIAXKAK26MY27JWTCO5
```

Comment:
bad idea to show credentials in a public repo 
replace by 

secret: ${S3_SECRET}
ccess: ${S3_ACCESS}